doc/README.md

```diff
@@ -490,6 +491,10 @@ to put it in the center of your view, and then
 This button resets the zoom level to 100%, the same as when you first load
 Cocreate.  This is helpful if you get lost zooming in or out.
 
+### <img src="icons/expand-arrows-alt.svg" width="18" alt="Zoom To Fit Icon"> Zoom To Fit
+
+This button will set the zoom and pan so that all elements fit inside the display.
```

Comment:
Maybe add something like "This can be useful to get an overall picture of the page, and then zoom into a part of interest. In particular, it can help you find parts of the page that are previously off-screen."

Reply:
Done!